app/routes.js

```diff
@@ -363,11 +364,13 @@ router.get('/stillbirth/task-list', function (req, res) {
     req.session.gestationWeeks = req.session.gestationWeeks
     req.session.mri = req.session.mri
     req.session.diagnosed = req.session.diagnosed
+    req.session.thoughtAlive = req.session.data['thought-alive']
```

Comment:
Unsure if this changes anything but the rhs here get's the data from the page that has the name 'thought-alive'. This get request is just getting what we declared on the event-type-3 page. Ignore if it doesn't make a difference